kymatio/scattering2d/backend/torch_skcuda_backend.py

```diff
@@ -11,7 +11,7 @@
 from ...backend.torch_backend import _is_complex
 from ...backend.torch_skcuda_backend import cdgmm
 
-@cupy.util.memoize(for_each_device=True)
+@cupy._util.memoize(for_each_device=True)
```

Comment:
(Next time, it's probably better to put this in a separate PR, but this is good for now.)